src/store/auth/operations/loginOperation.js

```diff
@@ -0,0 +1,40 @@
+import { createLogic } from 'redux-logic';
+import { LOGIN_SUBMIT, LOGIN_LOADING, LOGIN_ERROR, LOGIN_SUCCESS } from '../actions/types';
+import { API_KEY } from '../../../constants';
+
+export const loginOperation = createLogic({
+    type: LOGIN_SUBMIT,
+    latest: true,
+    async process({ action, axios }, dispatch, done) {
+        dispatch({type: LOGIN_LOADING, loading: true});
```

Comment:
do you plan to extract actions in to separate files? this could be very helpful

Reply:
added actions creators